test/jdk/java/nio/file/Path/PathOps.java

```diff
@@ -182,6 +183,19 @@ PathOps resolve(String other, String expected) {
         return this;
     }
 
+    // Note: "expected" is first parameter here
+    PathOps resolve(String expected, String first, String... more) {
+        out.format("test resolve %s varargs\n", path());
+        checkPath();
+        Path[] others = new Path[more.length];
+        int i = 0;
+        for (String s : more) {
+            others[i++] = Path.of(s);
+        }
+        check(path.resolve(Path.of(first), others), expected);
```

Comment:
This looks like it tests only the new method with Path elements.
Are there tests for the new default method that accepts String.

Reply:
Addressed in c2a8faaa1b81dbb40e1a5989a02c9e5935417358. Thanks for pointing it out.